src/components/relaying-educational/Footer/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Button } from '@mui/material'
+
+type ButtonProps = {
+  label: string
+  cb?: () => void
+}
+
+type FooterProps = {
+  back?: ButtonProps
+  next?: ButtonProps
+  justifyContent: 'space-between' | 'flex-start' | 'flex-end'
```

Comment:
Let's use the type here: `BoxProps['justifyContent']`.

Reply:
If you start from `flex-end`, you also need not pass this prop at all.

Reply:
What do you mean by 
> If you start from flex-end